Makefile

```diff
@@ -22,7 +22,7 @@ install:
 
 .PHONY: release
 release:
-	@go get github.com/goreleaser/goreleaser
+	@GO111MODULE=on go install github.com/goreleaser/goreleaser
```

Comment:
maybe just turn this on for the entire makefile?

Reply:
Well, we still run tests in travis w/out Go modules, to make sure that the project can be built either way. So turning it on in the Makefile would mean those CI jobs aren't doing what they're supposed to be doing.

At some point we can revisit dropping support for non-module builds. But that seemed more aggressive than the intended scope of this branch.